vernac/obligations.ml

```diff
@@ -20,6 +20,12 @@ open Pp
 open CErrors
 open Util
 
+(* EJGA: This should go away, no more need for fix_exn  *)
+module Stm = struct
+    let u = (fun x -> x)
+    let get_fix_exn () = u
+end
+
```

Comment:
In particular we should try to refactor the `undo_vernac_classifer` first (already marked as "CRUFT" in the source code), which should make error handling in the STM a bit clearer IMHO.